### DISCUSSION TITLE
How to reposition admin links?

### DISCUSSION THREAD

**Discussion by TUBii** [datenstrom/community#86] [Ask a question] [answered]
Hello! Thank You for creating Yellow! 

My question is: How to reposition admin links(Edit page, etc...)?

My goal is to put it somewhere lower, like after my <footer> tag. Its hiding my mobile navigation.
 I already tried to 'hack' core-webinterface.js but JS is not my strong point.

**Comment by markseuffert** [comment]
Thanks for your feedback! 

Can I have a look at the problem? Do you have a screenshot maybe?

**Comment by TUBii** [comment]
Here is the screenshot. 
![yellowadmin](https://cloud.githubusercontent.com/assets/1816440/8355986/ce4e232a-1b52-11e5-884c-0f0a88b95bf4.png)
I think its just need to place the code in different place. I found out its done by JS. But i cant deal with it.

**Comment by markseuffert** [comment, marked as answer]
Thanks, I see the problem. 

Usually the webinterface is added as first child to the `<body>` and followed by a couple of `<div>`s. I suggest to adjust the theme. For example position your mobile navigation relative to one of these divs. If you position something relative to an element in the page, instead of the body, then it won't overlap with the webinterface. What do you think?

**Comment by TUBii** [comment]
Good idea, i wil try it!
